README.md

```diff
@@ -5,22 +5,26 @@ OpenVPN for Android
 
 Description
 ------------
-With the new VPNService of Android API level 14+ (Ice Cream Sandwich) it is possible to create a VPN service that does not need root access. This project is a port of OpenVPN.
+In Android 4.0 (Ice Cream Sandwich) a VPN service can be made without root access, using the VPNService of API level 14+. \
+This project is a port of OpenVPN.
 
 <a href="https://f-droid.org/repository/browse/?fdid=de.blinkt.openvpn" target="_blank">
 <img src="https://f-droid.org/badge/get-it-on.png" alt="Get it on F-Droid" height="80"/></a>
 <a href="https://play.google.com/store/apps/details?id=de.blinkt.openvpn" target="_blank">
 <img src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge.png" alt="Get it on Google Play" height="80"/></a>
 
 Developing
----------------
-If you want to develop on ics-openvpn please read the [doc/README.txt](https://github.com/schwabe/ics-openvpn/blob/master/doc/README.txt) *before* opening issues or emailing me. 
+----------
+Read the [doc/README.txt](https://github.com/schwabe/ics-openvpn/blob/master/doc/README.txt) *before* opening issues or e-mailing. 
```

Comment:
Completely changes the meaning of this sentence.

Reply:
It is implied the user can e-mail about things other than things related to issues, but that doesn't need explaining.
With the *before*, the original reads ~"do you really have to do this", and ~"it is likely to annoy" to me.